grr/server/grr_response_server/gui/ui/components/expandable_hash/hashes/hashes.ng.html

```diff
@@ -0,0 +1,57 @@
+<div class="container mat-body-1">
+  <div *ngIf="!atLeastOneHashAvailable" class="noHashes">
+    no available hashes to display
+  </div>
+
+  <div *ngIf="atLeastOneHashAvailable" class="heading">
+    <button 
+      class="copy-button"
+      mat-icon-button
+      aria-label="Copy path to clipboard"
+      [cdkCopyToClipboard]="completeHashInformation">
```

Comment:
Sounds good. FYI Daniel White said that it would be useful for him if he could copy all hashes, possibly with the file path included. Related to that, Misha suggested that we might implement a more general copy-this-row functionality.